pkg/exchange/ftx/stream.go

```diff
@@ -60,6 +72,9 @@ func (s *Stream) Connect(ctx context.Context) error {
 	if err := s.ws.Connect(ctx); err != nil {
 		return err
 	}
+	s.ctx = ctx
+	s.isConnected = true
+	s.EmitStart()
```

Comment:
here you have the context, you can pass the context to the k-line worker

Reply:
The subscribeKLine() worker is not called in the Connect() function, (it is called in Subscribe()). Maybe we can move the query from subscribeKLine() to handleChannelKlineMessage() ?

Reply:
fixed in b7bf89f

Reply:
when user calls Subscribe() we will push the subscription to the subscription array.

we can start the workers when the Connect() method is called.